salt/api-gateway/init.sls

```diff
@@ -55,6 +55,10 @@ remove-old-kong-ppa:
     pkgrepo.absent:
         - name: deb https://dl.bintray.com/mashape/kong-ubuntu-trusty-0.9.x trusty main
 
+uninstall-old-kong:
+    pkg.purged:
+        - name: kong
```

Comment:
this would also stop the daemon (or fail horribly)

Reply:
it hasn't failed horribly for me yet, but I don't think we can achieve a zero-downtime upgrade to the new package name unless we have multiple nodes. I can add a `service.dead` state, but the `migrations up` command will also stop kong.

Reply:
as an aside, I issued a blanket `service nginx restart` from the salt master yesterday with no catastrophic effects. api-gateway definitely restarted it's nginx instance as it's now using the new ssl certificate. NewRelic didn't pick anything up